src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,14 @@
 ﻿#nullable enable
+Microsoft.Azure.StackExchangeRedis.AzureCacheOptions.CancellationToken -> System.Threading.CancellationToken
```

Comment:
Filter out any unnecessary lines, then move the remaining to PublicAPI.Shipped.txt